frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -235,6 +247,25 @@ export function BasicInformation({
               />
             </div>
           </div>
+          <div className="mt-5 p-3 border border-1 border-blue-gray-200 rounded-lg flex-col flex">
+            <h2 className=" text-lg mb-2">Register for Organiser Hub!</h2>
+            <p className="font-light text-sm">Join hundreds of sport societies hosting their events on Sportshub.</p>
+            <p className="font-light text-sm">
+              Leverage the ability to take bookings and payments right through the platform.
+            </p>
+            <button
+              className="ml-auto bg-black px-3 py-1.5 text-white rounded-lg mt-2"
+              type="button"
+              onClick={async () => {
+                setLoading(true);
+                window.scrollTo(0, 0);
+                const link = await getStripeStandardAccounLink(user.userId, "http://localhost:3000/organiser");
```

Comment:
we would need to replace localhost right?

Reply:
yes! thanks for the spot, let me replace it